### DISCUSSION TITLE
Multi Frame

### DISCUSSION THREAD

**Discussion by DaniEnsi** [TomSchimansky/CustomTkinter#99] [Q&A]
Hi, Ich habe versucht mehrere frames zu kreieren, habe es jedoch nicht geschafft... Ich habe mir ein video angeschaut, indem es erklärt wie man es im normalen Tkinter macht, aber sobaldd ich es ausgewechselt habe mit dem customtk, hat es mir nur noch errors gegeben...

:) Hilf mir bitte

**Comment by KAILO776** [comment]
Hallo, bitte zeigen Sie uns Ihre Codebeispiele und die aufgetretenen Fehler, damit wir Ihnen helfen können

Was Sie tun können, um mehrere Frames zu verwenden, ist pack() den Frame, den Sie verwenden möchten, und pack_forget() einmal den aktuellen Frame, wenn Sie wechseln

**Comment by felipetesc** [comment]
Here you can find 2 samples :https://felipetesc.github.io/CtkDocs/#/multiple_frames 

Update: There is a single sample for  5.2.0 now
